.pipelines/vhd-builder.yaml

```diff
@@ -44,6 +44,19 @@ phases:
       -e VHD_NAME=${VHD_NAME} \
       ${DEIS_GO_DEV_IMAGE} make az-copy
     displayName: Copying resource to Classic Storage Account
+    condition: not(${DRY_RUN})
```

Comment:
this ensures we only copy the disk to a classic storage account if `DRY_RUN` is set to `False` (`True` by default). That way, we won't clutter the classic SA for PR builds.